validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -117,25 +80,21 @@ def __init__(self, block_cache, state_view):
     def verify_block(self, block_state):
         return True
 
-    def compute_block_weight(self, block_state):
-        # longest chain wins
-        return block_state.block_num
-
 
 class ForkResolver(ForkResolverInterface):
-    # Provides the fork resolution interface for the BlockValidator to use
-    # when deciding between 2 forks.
+    """ DevMode ForkResolver Interface
```

Comment:
"DevMode ForkResolver implementation" (to stay consistent with other concrete classes in file).